test/index.js

```diff
@@ -55,6 +56,27 @@ describe('Log', () => {
         check({ additionalFields: null }, /^additionalFields must be an object$/);
     });
 
+    it('defaults to stdlogger', () => {
+
+        const server = new Hapi.Server();
+        server.register({ plugin: Plugin });
+        expect(server.registrations['@hapi/log']).to.exist();
```

Comment:
I'd change this expectation to:
```js
expect(server.registrations['@hapi/log'].log).to.be.an.instanceOf(Stdlogger);
```

IMO the current expectation only check that the plugin is properly registered that's all.

Reply:
I like your suggestion, but it doesn't look like the registration exports that as an option as it is made later:

```
{ version: '0.1.0', name: '@hapi/log', options: undefined }
```